hack/functest.sh

```diff
@@ -43,3 +43,30 @@ TEST_NAMESPACE=node-maintenance-operator go test ./test/e2e/... -root=$(pwd) -ku
 
 echo "E2e tests passed"
 
+echo "check validation of openaAPIV3Schema"
+
+$KUBECTL_CMD create -f _out/namespace-init.yaml
+
+$KUBECTL_CMD create -f _out/nodemaintenance_crd.yaml
```

Comment:
Does the test erase these on completion?

Reply:
added a trap to erase on completion.

Reply:
What about `go test` ?